frontend/src/app/modules/home/home-page/home-page.component.sass

```diff
@@ -204,10 +210,9 @@
 
 .home-content-wrp
     padding: 0
-    margin-left: 230px
     margin-top: 64px
+    margin-left: 230px
```

Comment:
Please, could you use sass variables for asside width and home-content-wrp padding for it.

Reply:
done